geps/gep-1897/index.md

```diff
@@ -49,6 +46,20 @@ this proposal.
 
 ## Non-Goals
 
+A major non-goal for this proposal is for configuration of the service mesh transport security.
+It is very common for service mesh implementations to implement some form of transparent transport security, whether that is WireGuard, mTLS, or others.
+This is completely orthogonal to the use cases being tackled by this GEP.
+
+* The "mesh transport security" is something invisible to the users application, and is simply used to secure communication between components in the mesh.
+* This proposal, instead, explicitly calls for sending TLS **to the users application**.
+
+Whether the service mesh implementation uses mTLS for it's implementation is not relevant to this proposal.
+A service mesh using mTLS can *additionally* originate TLS that is terminated by the users application (that is, two layers of TLS!).
```

Comment:
suggest adding what it means for Authz enforcement on L7 attributes (that are now encrypted and invisible to the mesh)

Reply:
I am worried about adding too much confusion here based on other comments where ~any service mesh comment is adding confusion